mafiasi/gprot/templates/gprot/index.html

```diff
@@ -66,4 +110,31 @@
     });
     $(".help-block").remove();
 </script>
+<script type="text/javascript">
+    function favorite(action){
+        let url = window.location.href
+        $.ajax({
+            type: 'POST',
+            url: '/gprot/favorite/',
+            headers: {'X-CSRFTOKEN': readCookie('csrftoken')},
+            dataType: 'text',
+            data: {'action': action, 'url': url},
+            success: function() {
+                location.reload();
+            }
```

Comment:
What happens when the action was not successful? Maybe an error message to the user? Or also a reload so that the user sees that the favorite was not saved.